python/tests/test_ops.py

```diff
@@ -1659,6 +1659,11 @@ def test_linspace(self):
         expected = mx.array(np.linspace(0, 1, 10))
         self.assertEqualArray(d, expected)
 
+        # Test num equal to 1
+        d = mx.linspace(1, 10, 1)
+        expected = mx.array(np.linspace(0, 1, 10))
+        self.assertEqualArray(d, expected)
```

Comment:
Does this test pass?

Reply:
Because it shouldn't 😄

Reply:
oops my bad, I forgot to make changes here

Reply:
done!